app/assets/sass/application.scss

```diff
@@ -399,14 +423,12 @@ tr.warning td:first-child{
 }
 
 .text-success {
-    color:#00823b;
-    padding-top:2px;
-    display:inline-block;
+    color: $govuk-success-colour
 }
 
 .text-success:before {
-    background-color: #00823b;
-    color:#00823b;
+    background-color:  $govuk-success-colour;
+    color: $govuk-success-colour
 }
```

Comment:
Not a comment on your update, just realised this rule and the one below are exactly the same 😁